cnd/src/bitcoin.rs

```diff
@@ -0,0 +1,26 @@
+//! This module is the home of bitcoin-specific types and functionality that is
+//! needed across several places in cnd.
+//!
+//! This involves:
+//!     - Creating wrapper types to allow for more ergonomic APIs of upstream
+//!       libraries
+//!     - Common functionality that is not (yet) available upstream
+
+use bitcoin_support::secp256k1;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
+pub struct PublicKey(bitcoin_support::PublicKey);
+
+impl PublicKey {
+    pub fn new(key: secp256k1::PublicKey) -> Self {
+        Self(bitcoin_support::PublicKey {
+            compressed: true,
```

Comment:
:question: Why is it compressed? Should there be a comment here explaining?

Reply:
That is there to indicate, how we want it to be serialized.

Reply:
I think it's fine. The current Bitcoin standard is to use compressed keys. Non-compressed keys are old school and unexpected.